File: src/Bank/Account.java
package src.Bank;

import java.util.Scanner;

public class Account {
    private double Balance = 0;
    private double Value;

    public void Balance() {
        if (Balance == 0) {
            System.out.println(Utilities.ANSI.BOLD + "Balance: " + Balance + Utilities.ANSI.RESET);
        } else if (Balance > 0) {
            System.out.println(Utilities.ANSI.BOLD_GREEN + "Balance: " + Balance + Utilities.ANSI.RESET);
        } else if (Balance < 0) {
            System.out.println(Utilities.ANSI.BOLD_RED + "Balance: " + Balance + Utilities.ANSI.RESET);
        } else {
            System.out.println(Utilities.ANSI.BOLD_RED + "FATAL ERROR: THE BALANCE COULD NOT BE LOADED!!!" + Utilities.ANSI.RESET);
        }
    }

    public void AddMoney(Scanner sc) {
        System.out.print(Utilities.ANSI.BOLD + "Add: " + Utilities.ANSI.RESET);

        if (sc.hasNextDouble()) {
            Value = sc.nextDouble();
            this.Balance += Value;
        } else {
            System.out.println("Invalid input. Please enter a valid number.");
            sc.next();
        }
    }

    public void ToTransferMoney(Scanner sc) {
        System.out.print(Utilities.ANSI.BOLD + "To transfer: ");

        if (sc.hasNextDouble()) {
            Value = sc.nextDouble();
            this.Balance -= Value;
        } else {
            System.out.println("Invalid input. Please enter a valid number.");
            sc.next();
        }
    }
}

File: src/account.rs
use std::io::{self, Write};

pub struct Account {
    balance: f32,
    value: f32,
}

impl Account {
    pub fn new(balance: f32, value: f32) -> Self {
        Self { balance, value }
    }

    pub fn balance(&self) -> f32 {
        self.balance
    }

    pub fn add_money(&mut self) {
        print!("Add: ");

        io::stdout().flush().expect("Error when use the flush!");

        let mut input: String = String::new();
        io::stdin()
            .read_line(&mut input)
            .expect("Error when read the value!");

        let value: f32 = input.trim().parse().expect("Type a valid number!");

        self.value = value;
        self.balance += value;
    }

    pub fn to_transfer(&mut self) {
        print!("To transfer: ");

        io::stdout().flush().expect("Error when use the flush!");

        let mut input: String = String::new();
        io::stdin()
            .read_line(&mut input)
            .expect("Error when read the value!");

        let value: f32 = input.trim().parse().expect("Type a valid number!");

        self.value = value;
        self.balance -= value;
    }
}
